TrackParamFilter/sections/specific.txt

```diff
@@ -11,6 +11,8 @@
 ||go.redirectingat.com^$removeparam=xcust
 ||go.redirectingat.com^$removeparam=sref
 ||go.redirectingat.com^$removeparam=id
+! https://github.com/AdguardTeam/AdguardFilters/issues/149710
+||klook.com$removeparam=aid
```

Comment:
should be separator after the domain and before special character
`||klook.com^$removeparam=aid`

Reply:
Could you please also add:
```
||klook.com^$removeparam=clickId
||klook.com^$removeparam=spm
```
For links such as:
https://www.klook.com/zh-TW/?spm=Activity.TopNavigation.BackHome&clickId=a217dd6556

Reply:
@TobySkarting  hello? any news?